### DISCUSSION TITLE
Log borg commands to log files with verbosity

### DISCUSSION THREAD

**Discussion by cjustin88** [borgbase/vorta#844] [Ideas]
**Is your feature request related to a problem? Please describe.**
Please consider adding a feature to save the output of `borg` commands to log files, with `--debug` turned on.

I'm trying to debug why Vorta suddenly closed during a backup. I suspect the problem is in Vorta, not in borg. If I had a log of what each `borg` command was doing, I could confirm whether the command finished successfully or not. The last line of `vorta.log` just says that the borg command was issued:

```vorta.borg.borg_thread - INFO - Creating archive at```

**Describe the solution you'd like**
Add option to save the output of `borg` commands to log files with verbosity. Perhaps this can be added to the Misc panel?

e.g. if feature is turned on, the command that would be issued would be:

```vorta.borg.borg_thread - INFO - Running command /usr/bin/borg create --debug ...```

saving the stdout and stderr to `/home/user/.cache/Vorta/log/timestampXXX_borg.log", for example.

**Describe alternatives you've considered**

I tried to find where --log-json is writing to, but I think it is being written to stdout and intercepted by Vorta...so I can't independently tail the borg log...?

Thanks.

**Comment by cjustin88** [comment]
Similar request here https://github.com/borgbase/vorta/issues/282 but I don't see any output from borg being written into `vorta.log`.

**Comment by samuel-w** [comment]
Try running Vorta from the command line to get the exception, exceptions not being logged will be fixed by #628 .

Errors in borg should not crash vorta, so its more likely a vorta problem

**Comment by cjustin88** [comment]
@samuel-w Thanks, I will try that. It would still be useful to be able to configure borg verbosity and have the logs written out. I tried adding `--debug` as an extra borg command line option when adding a repository, but nothing more is printed to `vorta.log` (just `vorta.borg.borg_thread - INFO - Running command /usr/bin/borg create --debug ...`).

**Comment by samuel-w** [comment]
To read out the logs, we would have to print the entire json output. For something like `borg create`, it would lead to a massive log file, since each file gets its own line and so does each change in size of the backup.

**Comment by cjustin88** [comment]
It would be good to at least have the option. Large log files should not surprise the user if the user chooses to explicitly turn on debug.
